MockKit/MockPumpManagerState.swift

```diff
@@ -9,6 +9,13 @@
 import Foundation
 
 public struct MockPumpManagerState {
+    public enum DeliverableIncrements: String, CaseIterable {
+        case omnipod
+        case medtronicX22
+        case medtronicX23
+    }
```

Comment:
[nit] just wondering if there should be some sort of visual cue about the type of simulator you have selected. Currently the simulator image in Loop settings is always a Medtronic look-a-like. Should it switch to a pod look-a-like? For the Medtronic variant, should it be overlaid with the product ID?

Reply:
Great idea for a future improvement!
We'd need to implement some way to indicate to Loop that the pump manager's image/name has changed to reload the settings row showing the simulator, since right now they're assumed to be static. I'll leave this for another day since the settings screen is being redone.